src/rtpghi~.c

```diff
@@ -110,8 +91,9 @@ void *rtpghi_tilde_new(t_symbol *s, int argc, t_atom *argv)
     
     post("%p", (x->sta_pd));
     int init_s = 0;
-    
-    init_s = phaseret_rtpghi_init_s(gamma, W, a , M, tol, do_causal, &(x->sta_pd));
+	ltfat_int w = 1;
+	
+    init_s = phaseret_rtpghi_init_s(gamma, w, a , M, tol, do_causal, &(x->sta_pd));
```

Comment:
you might want to call `phaseret_rtpghi_init_s` only if the parameters have changed, so that when you turn the DSP off and then on, the state is preserved (the DSP might be restarted for whatever reasons and i guess that re-initializing the phaseret_state will cause a click)